x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -13201,4 +13200,4 @@
     "xpack.watcher.watchEdit.thresholdWatchExpression.aggType.fieldIsRequiredValidationMessage": "フィールドを選択してください。",
     "xpack.watcher.watcherDescription": "アラートの作成、管理、監視によりデータへの変更を検知します。"
   }
-}
+}
```

Comment:
should the newline be here? same as below

Reply:
I think it should. Kibana's .editorconfig has `insert_final_newline = true` and Prettier puts it there by default. If there's supposed to be an exception here it should be designated by the linter to be so.